deploy/environments/applications.yaml

```diff
@@ -23,7 +23,7 @@ spack:
     - brion@3.3.1 +python
     - circuit-build
     - connectome-tools@0.3.3
-    - emsim ^brion@3.1.0
+    - emsim@1.0.1
```

Comment:
```suggestion
    - emsim
```
Leave the version number off, it'll always deploy just the latest version.

Reply:
that's preferable? If so, cool w/ me